src/demos/cern-videos/Results.js

```diff
@@ -54,7 +54,7 @@ export class Results extends Component {
           </Grid.Column>
         </Grid>
         <Grid relaxed style={{ padding: '2em 0' }}>
-          <ResultsMultiLayout overridableId="cernvideos" />
+          <ResultsMultiLayout overridableId='cernvideos' />
```

Comment:
I am not sure if you are using prettier already but I think normally its double quotes in the props and single in imports (to be verified)